backend/clubs/models.py

```diff
@@ -1846,6 +1846,7 @@ class Ticket(models.Model):
     holding_expiration = models.DateTimeField(null=True, blank=True)
     carts = models.ManyToManyField(Cart, related_name="tickets", blank=True)
     price = models.DecimalField(max_digits=5, decimal_places=2)
+    transferrable = models.BooleanField(default=True)
```

Comment:
Let's add an option to make tickets non-transferrable at event/ticket creation. Would be nice to display this as a frontend toggle that admins can opt into.